repo/wallet/data.go

```diff
@@ -4,14 +4,14 @@ import (
 	"io"
 	"time"
 	"bytes"
-	"image/jpeg"
+	_ "image/jpeg"
 	_ "image/png" // register other possible image types for decoding
 	_ "image/gif"
 	"encoding/json"
-	"image"
+	_ "image"
```

Comment:
apparently it's bad practice to use the underscore method import unless you are just "loading codecs" - if you dont need these just delete em

Reply:
done